client/src/app/site/common/components/user-statistics/user-statistics.component.ts

```diff
@@ -105,6 +112,20 @@ export class UserStatisticsComponent extends BaseComponent {
         }
     }
 
+    public getModelRequest(): SimplifiedModelRequest {
+        return {
+            viewModelCtor: ViewMeeting,
+            ids: [this.activeMeetingId],
+            /**
+             * TODO: Something is not right. This throws:
+             * "Relation with ownIdField speaker_ids (effective speaker_ids) in collection meeting unknown!"
+             * tho it should not
+             */
+            follow: [{ idField: 'speaker_ids' }],
+            // fieldset: []
```

Comment:
@FinnStutzenstein

Reply:
Maybe it would be:

```suggestion
            viewModelCtor: ViewMeeting,
            ids: [this.activeMeetingId],
            follow: [{ idField: 'speaker_ids_ids' }],
            // fieldset: []
```

Then you have to update the `repositories/relations.ts` line 337 to:

```ts
    ...makeM2O({
        OViewModel: ViewMeeting,
        MViewModel: ViewSpeaker,
        OField: 'speakers',
        MField: 'meeting'
    }),
```

Reply:
Yes, `speaker_ids` is wrong there.